### PR TITLE
fuzz: use the right kvm-ioctls version

### PR DESCRIPTION
**Pull request by alyssais**
This fixes building for fuzzing on aarch64.

### CODE REVIEW

**Review by rbradford — COMMENTED** [review]
We try and keep the versions inside the fuzz workspace consistent with main workspace. It's possible it just needs a Cargo.lock bump for both.

**Comment by alyssais** [comment]
This is _making_ the fuzz workspace consistent with the main one.

File: fuzz/Cargo.toml [hunk 1]
```diff
@@ -35,6 +35,7 @@ path = ".."
 
 [patch.crates-io]
 kvm-bindings = { git = "https://github.com/cloud-hypervisor/kvm-bindings", branch = "ch-v0.6.0-tdx" }
+kvm-ioctls = { git = "https://github.com/rust-vmm/kvm-ioctls", rev = "23a3bb045a467e60bb00328a0b13cea13b5815d0" }
```
Comment (rbradford) [inline fuzz/Cargo.toml:38]: The main workspace has it like this:

`kvm-ioctls = { git = "https://github.com/rust-vmm/kvm-ioctls", branch = "main" }`

Comment (rbradford) [reply fuzz/Cargo.toml:38]: I think we should tackle this as part of our bulk upgrade in #6072

Comment (alyssais) [reply fuzz/Cargo.toml:38]: Okay, so we close this?

Comment (rbradford) [reply fuzz/Cargo.toml:38]: Yes, lets do that - thanks for bringing it to our attention. To check that I remember to update this when do the big version update.

**Pull request closed without merging**